cpp/include/Ice/InputStream.h

```diff
@@ -24,6 +24,14 @@
 #include <string>
 #include <string_view>
 
+// ICE_UNALIGNED means the platform is little endian and supports unaligned reads.
```

Comment:
In other words, if you want this ICE_UNALIGNED behavior for your app on ARM64, you just need to define ICE_UNALIGNED when you build your app.

Maybe we should always leave it undefined by default, and only test it in the Ice/customs test?